addons/mail/static/src/js/chat_manager.js

```diff
@@ -289,10 +293,12 @@ function make_channel (data, options) {
         channel.type = data.public !== "private" ? "public" : "private";
     }
     if (_.size(data.direct_partner) > 0) {
+        var last_seen_on = data.direct_partner[0].last_seen;
```

Comment:
variables should be camelcased (see coding guidelines: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines). so, last_seen_on should be instead lastSeenOn